directory/src/main/java/org/ecclesia/directory/api/Organization.java

```diff
@@ -0,0 +1,11 @@
+package org.ecclesia.directory.api;
+
+public class Organization {
+
+  public
+    void
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-5/directory/src/main/java/org/ecclesia/directory/api/Organization.java L6

&#39;void&#39; has incorrect indentation level 4, expected level should be 6.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1682716480>*